pathway-analysis/ora_with_webgestaltr.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "Over-representation analysis with WebGestaltR"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: J. Taroni for ALSF CCDL
+date: 2019
+---
+
+## Background
+
+Over-representation analysis (ORA) is a method of pathway or gene set analysis
+where one can ask if a set of genes (e.g., those differentially expressed
```

Comment:
```suggestion
where one can ask if a set of genes (e.g. those differentially expressed
```

Reply:
I googled it and found:
> In general, you add a comma after e.g. and between each subsequent example if there is more than one item in your list.

[ref](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/)